src/components/primitive/drop-down/drop-down.component.tsx

```diff
@@ -0,0 +1,19 @@
+import { DropDownComponentProps, DropDownOptionProps } from './drop-down.component.props';
+import { DropdownWrapper, StyledLabel, StyledSelect, StyledOption } from './drop-down.component.styles';
+
+export const DropDown: React.FC<DropDownComponentProps> = (props) => {
+  const { label, children, ...rest } = props;
+
+  return (
+    <DropdownWrapper {...rest}>
+      <StyledLabel>{label}</StyledLabel>
+      <StyledSelect>{children}</StyledSelect>
+    </DropdownWrapper>
+  );
+};
+
+export const Option: React.FC<DropDownOptionProps> = (props) => {
```

Comment:
Don't export this. Options must be rendered within the dropdown component. It's not obvious to consumers of dropdown component that Option must be imported. Instead, make options[] as a prop.

Reply:
yeah, removed in the latest commit.